lib/executor/common_test.go

```diff
@@ -0,0 +1,55 @@
+package executor
+
+import (
+	"context"
+	"io/ioutil"
+	"math/rand"
+	"testing"
+
+	"github.com/loadimpact/k6/lib"
+	"github.com/loadimpact/k6/lib/testutils"
+	"github.com/loadimpact/k6/stats"
+	"github.com/sirupsen/logrus"
```

Comment:
Let do group import.

Reply:
I'm not sure what you mean, sorry.

These imports weren't written by hand, they're handled by `goimports`.

Reply:
We should:

 - Stdlib
 - third party libs
 - k6 libs

`goimports` does not handle this by default, you should:

```
goimports -local github.com/loadimpact/k6
```

Most editor let you config this.

Reply:
I don't think this ... adds any real value ... And will be even harder to enforce on people doing contributions, as such I vote against this being enforced (aka you can do it, but I don't think we need to)
cc @na--

Reply:
@MStoykov That's why I said we *should*, most of the projects I have been working on follow this, that actually help with reading the code. See the original issue in https://github.com/golang/go/issues/12420

Reply:
I'm fine with this approach, and was used to it in Python projects. Agree that it makes reading imports easier.

The only drawback is that it can't be easily enforced for external contributions (maybe we could fix it for them in `make check`?), so it won't be consistent. But in this case as long as internal contributors are doing it, we can benefit from it. (Also see conventional commits... ;) Maybe we should mention it in CONTRIBUTING.md, though I doubt that is widely read.

I added it to my Emacs config, and will push the changes to files touched here. Now I just need to make it variable per project... ':D

Reply:
I dont think it makes external contribution harder. You code Go, then you’re using goimports, it’s only adding argument. And as I said, it’s just one config in most editors

Reply:
See eda02aa1.